lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -6048,13 +6049,16 @@ void TypeChecker::checkConformancesInContext(DeclContext *dc,
           auto insertionLoc =
             classDecl->getAttributeInsertionLoc(/*forModifier=*/false);
           if (isFixable) {
+            Mangle::ASTMangler mangler;
```

Comment:
Can you leave behind a comment noting that this is deliberately using the Swift 3 mangling?